wp-content/themes/bpr2018/resources/assets/sass/_front_page.scss

```diff
@@ -3,6 +3,7 @@
 
   .post-title-large { margin-bottom: $spacing-sm; }
   .post-title-small { margin: $spacing-xs 0 $spacing-xs 0; }
+  .title-width-limiter { width: 25vw; }
```

Comment:
I think you could directly apply `width: 25vw` to `.post-title-small`. `.title-width-limiter
` looks unecessary to me.